### DISCUSSION TITLE
Welcome to Sandwich Discussions!

### DISCUSSION THREAD

**Discussion by skydoves** [skydoves/sandwich#9] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hgoda** [comment]
I need example app sample how to use 
with api only

**Reply by skydoves** [reply]
Here is the [demo](https://github.com/skydoves/Sandwich/tree/master/app) that used API only.

**Comment by Khalid-Enabil** [comment]
@skydoves 
Thank you for your efforts.
Would like to see a demo - an example that uses the function
`public suspend inline fun <T, V> ApiResponse<T>.suspendOnSuccess(`
because I didn't catch up everything inside, can this method get the success with a failure response, because in the API I have I send the response :{
"success": "false"
"data": "["error here"]"
}
in addition, the error may return as a string only. 
Need more illustration because I think I use it incorrectly and there are more ways that can be used of.

**Comment by johngray1965** [comment]
Thank you for this.  The design looks great.
